src/Core/Plugin.php

```diff
@@ -186,10 +186,14 @@ protected static function _loadConfig()
         if (Configure::check('plugins')) {
             return;
         }
-        try {
-            Configure::load('plugins');
-        } catch (\Exception $e) {
+
+        $vendorFile = ROOT . DS . 'vendor' . DS . 'cakephp-plugins.php';
```

Comment:
In that case conditionally check on that constant and automatically fallback to hardcoded 'vendor' maybe?